### ISSUE TITLE
no tf_efficientnet_b3_ns_s8 model in geffnet

### ISSUE THREAD

**Issue by cv-lab-x** [tusen-ai/Anchor3DLane#5]
hi, thanks for your great work!
there's no tf_efficientnet_b3_ns_s8 model in geffnet, only tf_efficientnet_b3_ns,   [](https://github.com/rwightman/gen-efficientnet-pytorch)

looking forward to your reply, thanks @spyflying

**Comment by spyflying** [comment]
Sorry for our previous negligence. We have made several modifications to the original efficientnet for task adaptation. By adjusting the downsampling strides, we obtain tf_efficientnet_b3_ns_s8 based on the original tf_efficientnet_b3_ns. We have included the modified geffnet module. Please re-clone the updated repo.

**Comment by cv-lab-x** [comment]
> Sorry for our previous negligence. We have made several modifications to the original efficientnet for task adaptation. By adjusting the downsampling strides, we obtain tf_efficientnet_b3_ns_s8 based on the original tf_efficientnet_b3_ns. We have included the modified geffnet module. Please re-clone the updated repo.

it works, thanks for your quick reply!  
Another question: how did you create the data_lists/traning.txt and data_lists/validation.txt, cause there're not data_lists folder in openlane data, and i found that can not find some files in openlane datasets  in validation.txt  eg: validation/segment-271338158136329280_2541_070_2561_070_with_camera_labels/150869126825189000  

@spyflying

**Comment by spyflying** [comment]
The data lists are generated based on the processed pickle files, i.e., the `training.txt' includes files with 1 or more lanes in the training set, and the `validation.txt' includes all the files in the validation set. You can freely generate the `training.txt'  by collecting all the file names in your cache_dense/training folder. 

For the missing validation file, I guess it may be due to the different versions of Openlane. This file is not included in the annotations of Openlane v1.2 compared with the original Openlane annotations. You can remove this line from `validation.txt' and I think there should be little difference in the validation results.
